src/components/common/ButtonWithIcon.vue

```diff
@@ -33,6 +38,29 @@ export default {
       type: String,
       default: '',
     },
+  },
+  computed: {
+    classes() {
+      return `${this.disabled ? 'disabledIcon' : this.type + 'Icon'} ${this.size}}`
+    },
   }
 }
 </script>
+
+<style scoped>
+.primaryIcon{
```

Comment:
I wouldn't create new class names like this. You can just use `.primary svg` as we are already assigning it in this way. See more here: https://github.com/Galleme/frontend/issues/28#issuecomment-917533657